### ISSUE TITLE
[BUG] Setting port for environmental variable doesn't change the port

### ISSUE THREAD

**Issue by WeebHiroyuki** [kyb3r/modmail#2912]
**Describe the bug**
Setting the port in the `.env` file doesn't change the actual port.

**Bot Info**
Self-hosted on Debian 10, Python 3.7

**To Reproduce**
Steps to reproduce the behaviour:
1. Who can reproduce (ex. anyone, owners)? anyone
2. Where can it be reproduced (ex. in thread channels, recipient DM's)? thread logs channel
3. Done what to cause the error? Open a thread and close it. Get the log URL from that.
4. Any recently made changes to your bot? No.
5. Errored. N/A

**Error Logs**
If your Modmail bot is online, type `@modmail debug hastebin` and include the link here.
If your Modmail bot is not online or the previous command did not generate a link, do the following:

1. Select your *bot* application at https://dashboard.heroku.com
2. [Restart your bot](https://i.imgur.com/3FcrlKz.png)
3. Reproduce the error to populate the error logs
4. [Copy and paste the logs](https://i.imgur.com/TTrhitm.png)

**Screenshots**
Add screenshots to help explain your problem.

**Additional context**
Basically, the port isn't being set as it should (from the `.env` file).

**Comment by coleslaw12** [comment]
Im confused as to what the error is. There isnt an error listed, and the “done what to cause an error” section is able to be done for me.

**Comment by WeebHiroyuki** [comment]
Hi, I'm sorry about this. I just realised that the logs aren't hosted in this repository. I'll close this issue now.
